tests/Latte/Parser/VarType/Fixture/var_type.php.inc

```diff
@@ -0,0 +1,25 @@
+{varType Rector\Nette\Tests\Rector\Latte\RenameMethodLatteRector\Source\SomeClass $abcd}
+{varType string $text}
+{varType int $number}
+-----
+Array
+(
+    [0] => Rector\Nette\ValueObject\LatteVariableType Object
+        (
+            [name:Rector\Nette\ValueObject\LatteVariableType:private] => abcd
+            [type:Rector\Nette\ValueObject\LatteVariableType:private] => Rector\Nette\Tests\Rector\Latte\RenameMethodLatteRector\Source\SomeClass
+        )
+
+    [1] => Rector\Nette\ValueObject\LatteVariableType Object
+        (
+            [name:Rector\Nette\ValueObject\LatteVariableType:private] => text
+            [type:Rector\Nette\ValueObject\LatteVariableType:private] => string
+        )
+
+    [2] => Rector\Nette\ValueObject\LatteVariableType Object
+        (
+            [name:Rector\Nette\ValueObject\LatteVariableType:private] => number
+            [type:Rector\Nette\ValueObject\LatteVariableType:private] => int
+        )
+
+)
```

Comment:
Wow, this is amazing :clap: easy to read, clear and easy to extend

Reply:
Thanks again :) I thought you will not like it O:-)